src/views/v2/month/calendar-body/day.php

```diff
@@ -108,7 +120,16 @@ class="tribe-events-calendar-month__day-cell tribe-events-calendar-month__day-ce
 	>
 		<h3 class="tribe-events-calendar-month__day-date tribe-common-h4">
 			<span class="tribe-common-a11y-visual-hide">
-				<?php echo esc_html( sprintf( _n( '%s event', '%s events', $day['found_events'], 'the-events-calendar' ), number_format_i18n( $day['found_events'] ) ) ); ?>,
+				<?php
+				echo esc_html(
+					sprintf(
+						_n( '%1$s %2$s', '%1$s %3$s', $day['found_events'], 'the-events-calendar' ),
+						number_format_i18n( $day['found_events'] ),
+						$events_label_singular,
+						$events_label_plural
+					)
+				);
+				?>,
```

Comment:
:no_entry_sign: Closing PHP tag must be on a line by itself

<sub>hash: 326b9be8b46f9d7b1a6f6a2000bce91f</sub>